__tests__/git.test.ts

```diff
@@ -191,17 +191,11 @@ describe('git', () => {
         }
       })
 
-      try {
-        await init(action)
-      } catch (e) {
-        expect(execute).toBeCalledTimes(0)
-        expect(e.message).toMatch(
-          "There was an error initializing the repository: Incorrectly formatted build folder. The deployment folder cannot be prefixed with '/' or './'. Instead reference the folder name directly. ❌"
-        )
-      }
+      await init(action)
+      expect(execute).toBeCalledTimes(6)
     })
 
-    it('should fail if the build folder begins with a ./', async () => {
+    it('should not fail if the build folder begins with a ./', async () => {
```

Comment:
Same goes for the other tests in this suite that perform the same check. These were checking explicitly for an error state previously but as that state/check is gone there's no real need for them to stick around.